### DISCUSSION TITLE
Submission: Simulation.log & Synthesis report

### DISCUSSION THREAD

**Discussion by b08901102** [bol-edu/HLS-SOC-Discussions#115] [General]
在Workbook中有提到應該繳交的檔案中包含Simulation.log和Synthesis report，但由於在synth_1、project.sim、甚至是執行makefile後都可以找到許多相似的.log和.rpt檔，所以在此想詢問助教究竟哪些檔案是需要被繳交的？抑或是沒有硬性規定呢？

**Comment by ZheChen-Bill** [comment]
基本上你有執行的步驟都會產生相對應的file，你需要繳交simulate log (simulation所輸出的檔案)，utilization report , synthesis report(synthesis產生的檔案)，以及 timing report(report time summary)。
